components/catalogs/front/src/components/ProductSelection/ProductSelection.tsx

```diff
@@ -38,6 +40,7 @@ const ProductSelection: FC<Props> = ({criteria, onChange, errors}) => {
     return (
         <ProductSelectionContext.Provider value={dispatch}>
             <Header>
+                <CountSelectedProductsByCriteria criteria={debouncedValue} />
```

Comment:
```suggestion
                <CountSelectedProductsByCriteria criteria={debouncedCriteria} />
```
`criteria={debouncedValue}` is hard to understand IMO